includes/class-wc-checkout.php

```diff
@@ -388,12 +388,30 @@ public function create_order( $data ) {
 			// Save the order.
 			$order_id = $order->save();
 
+			/**
+			 * Action hook fired after an order is created used to add custom meta to the order.
+			 *
+			 * @since 3.0.0
+			 */
 			do_action( 'woocommerce_checkout_update_order_meta', $order_id, $data );
 
+			/**
+			 * Action hook fired after an order is created.
+			 *
+			 * @since 4.1.0
```

Comment:
Should these be updated to 4.3?

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/26395/commits/b8a1a69b2b41c9cde539ff2a223cc35fa3d8714e